localisation/english/iisc_l_english.yml

```diff
@@ -165,27 +165,27 @@
- iisc_registry.1.desc: "Welcome to the corporate registry! Please select the corporation you wish to interact with."
- iisc_registry.1.corp_civic_0: "[iisc_registry_corp_0.GetCorpCivicIcon]"
+ iisc_registry.1.desc: "Welcome to the corporate registry!\nSelect the corporation you wish to interact with."
+ iisc_registry.1.corp_civic_0: "[iisc_registry_corp_0.GetCorpCivicIcon] [iisc_registry_corp_0.GetCorpEthicIcon]"
  iisc_registry.1.corp_civic_tooltip_0: "[iisc_registry_corp_0.GetCorpCivicTooltip]"
  iisc_registry.1.corp_name_0: "[iisc_registry_corp_0.GetName]"
- iisc_registry.1.corp_hq_0: "[iisc_registry_hq_0.GetName] [iisc_registry_hq_0.GetPlanetTypeIcon]"
- iisc_registry.1.corp_civic_1: "[iisc_registry_corp_1.GetCorpCivicIcon]"
+ iisc_registry.1.corp_hq_0: "[iisc_registry_hq_0.GetPlanetTypeIcon] [iisc_registry_hq_0.GetName]"
+ iisc_registry.1.corp_civic_1: "[iisc_registry_corp_1.GetCorpCivicIcon] [iisc_registry_corp_1.GetCorpEthicIcon]"
  iisc_registry.1.corp_civic_tooltip_1: "[iisc_registry_corp_1.GetCorpCivicTooltip]"
  iisc_registry.1.corp_name_1: "[iisc_registry_corp_1.GetName]"
- iisc_registry.1.corp_hq_1: "[iisc_registry_hq_1.GetName] [iisc_registry_hq_1.GetPlanetTypeIcon]"
- iisc_registry.1.corp_civic_2: "[iisc_registry_corp_2.GetCorpCivicIcon]"
+ iisc_registry.1.corp_hq_1: "[iisc_registry_hq_1.GetPlanetTypeIcon] [iisc_registry_hq_1.GetName]"
+ iisc_registry.1.corp_civic_2: "[iisc_registry_corp_2.GetCorpCivicIcon] [iisc_registry_corp_2.GetCorpEthicIcon]"
  iisc_registry.1.corp_civic_tooltip_2: "[iisc_registry_corp_2.GetCorpCivicTooltip]"
  iisc_registry.1.corp_name_2: "[iisc_registry_corp_2.GetName]"
- iisc_registry.1.corp_hq_2: "[iisc_registry_hq_2.GetName] [iisc_registry_hq_2.GetPlanetTypeIcon]"
- iisc_registry.1.corp_civic_3: "[iisc_registry_corp_3.GetCorpCivicIcon]"
+ iisc_registry.1.corp_hq_2: "[iisc_registry_hq_2.GetPlanetTypeIcon] [iisc_registry_hq_2.GetName]"
+ iisc_registry.1.corp_civic_3: "[iisc_registry_corp_3.GetCorpCivicIcon] [iisc_registry_corp_3.GetCorpEthicIcon]"
  iisc_registry.1.corp_civic_tooltip_3: "[iisc_registry_corp_3.GetCorpCivicTooltip]"
  iisc_registry.1.corp_name_3: "[iisc_registry_corp_3.GetName]"
- iisc_registry.1.corp_hq_3: "[iisc_registry_hq_3.GetName] [iisc_registry_hq_3.GetPlanetTypeIcon]"
- iisc_registry.1.corp_civic_4: "[iisc_registry_corp_4.GetCorpCivicIcon]"
+ iisc_registry.1.corp_hq_3: "[iisc_registry_hq_3.GetPlanetTypeIcon] [iisc_registry_hq_3.GetName]"
+ iisc_registry.1.corp_civic_4: "[iisc_registry_corp_4.GetCorpCivicIcon] [iisc_registry_corp_4.GetCorpEthicIcon]"
  iisc_registry.1.corp_civic_tooltip_4: "[iisc_registry_corp_4.GetCorpCivicTooltip]"
  iisc_registry.1.corp_name_4: "[iisc_registry_corp_4.GetName]"
- iisc_registry.1.corp_hq_4: "[iisc_registry_hq_4.GetName] [iisc_registry_hq_4.GetPlanetTypeIcon]"
+ iisc_registry.1.corp_hq_4: "[iisc_registry_hq_4.GetPlanetTypeIcon] [iisc_registry_hq_4.GetName]"
  iisc_registry.1.foreign_list: "View Foreign Corporations"
```

Comment:
These same changes need to be made to the French localisation file, if I am not mistaken?

Reply:
Oof good point